### PR TITLE
feat: allow pass '--quiet' when building.

### PR DESCRIPTION
**Pull request by zhyd1997**
Resolves: #40

### CODE REVIEW

**Comment by pascalgrimaud** [comment]
any chance to have this @Gnuk plz?

File: src/assets-build.ts [hunk 1]
```diff
@@ -29,8 +29,10 @@ if (!fs.existsSync(projectDist)) {
 
 const manageCopy = (...copyargs: any[]) => copy(...copyargs)
   .on(
-    copy.events.COPY_FILE_COMPLETE,
-    (copyOperation: any) => console.info(`${copyOperation.src} => ${copyOperation.dest}`),
+    (copyOperation: any) => {
+      if (!process.argv[2]) return
```
Comment (Gnuk) [inline src/assets-build.ts:33]: Maybe you can try to inject the augment instead of have this as a side effect (same for next parts)

**Review by Gnuk — CHANGES_REQUESTED** [review]
Maybe it's possible to add a test to check there are no logs in quiet mode.

File: src/assets-build.ts (hunk 1 above)
Comment (zhyd1997) [reply src/assets-build.ts:33]: Okay, I will take a look!

Thanks for your quick review!

**Comment by Gnuk** [comment]
> any chance to have this @Gnuk plz?

Yes maybe but I also don't know if it's better to have a concise log in the normal way and a verbose mode to have details.

**Comment by Gnuk** [comment]
Done with #137

**Pull request closed without merging**